setup.py

```diff
@@ -37,6 +37,7 @@
             'checkcommit = openwisp_utils.qa:check_commit_message',
         ],
     },
+    scripts=["openwisp-utils-qa-checks"],
```

Comment:
please use single quotes (`'`) for consistency, keep in mind to always be consistent with the styles already used in the program